lib/meson.build

```diff
@@ -78,6 +78,7 @@ endif
 if install_headerfiles
 	install_headers([
 		'tinytexexport.h',
-		'tinytex.h'
+		'tinytex.h',
+		'tinytexconfig.h'
```

Comment:
This fails, as tinytexconfig.h does not exist. You need to assign a lside variable from the configure_file that configures the `tinytexconfig.h.in` and put the var in the install_headers array